src/client.js

```diff
@@ -3,7 +3,8 @@ window.qed = {
   Editor: require('./editor'),
   Point: require('./point'),
   Range: require('./range'),
-  Toolbar: require('./toolbar')
+  Toolbar: require('./toolbar'),
+  StemTracker: require('./stem-tracker')
```

Comment:
let's put a few blank newlines above here and a comment e.g.

// extras or "plugins" - these might be moved into a separate repo in the future